### PR TITLE
[NCL-6194] Add functionality to process the results of Deliverable Analysis

### PR DESCRIPTION
**Pull request by janinko**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/pnc/wiki/Changelog) for your change if user-facing?
* [x] Have you added unit tests for your change?

### CODE REVIEW

File: facade/src/main/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerResultProcessor.java [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    @Inject
+    private TargetRepositoryRepository targetRepositoryRepository;
+
+    /**
+     *
+     * @param milestoneId
+     * @param builds
+     * @param distributionUrl
+     */
+    public void processBuild(int milestoneId, Collection<Build> builds, String distributionUrl, User user) {
```
Comment (jbartece) [inline facade/src/main/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerResultProcessor.java:67]: Add javadoc for this method, please.

**Review by jbartece — CHANGES_REQUESTED** [review]
Hi, it looks good, but I miss at least one javadoc on the processBuild method and some high level tests at least for this method.

File: facade/src/main/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerResultProcessor.java (hunk 1 above)
Comment (janinko) [reply facade/src/main/java/org/jboss/pnc/facade/deliverables/DeliverableAnalyzerResultProcessor.java:67]: ah, I inteded to add javadoc as you cen see but then somehow forgot about it

**Comment by jbartece** [comment]
@janinko I see that you force-pushed it, but I don't see any of those issues fixed.

**Comment by janinko** [comment]
> @janinko I see that you force-pushed it, but I don't see any of those issues fixed.
That was fixing compiling issue. I'm still working on tests.

**Comment by janinko** [comment]
@jbartece tests and javadoc added

**Review by jbartece — APPROVED** [review]

**Comment by janinko** [comment]
retest this please

**Pull request merged**